Fix options example (#390)
Laravel-Backpack/docs: 5.x/crud-fields.md

    // most options provided by intlTelInput.js are supported, you can try them out using the `config` attribute;
    //  take note that options defined in `config` will override any default values from the field;
    'config' => [
        'onlyCountries' => ['bd', 'cl', 'in', 'lv', 'pt', 'ro'],

>         'initialCountry' => 'cl', // if restrict country with onlyCountries option need to be in list to work

Reviewer: ```suggestion
        'initialCountry' => 'cl', // this needs to be in the allowed country list, either in `onlyCountries` or NOT in `excludeCountries`
```
